source/2019-12-13-countdown-to-the-new-year-ember-simple-auth.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year - Ember Simple Auth
+author: Samanta de Barros
+tags: 2019, Addons, DecEmber
+alias: 2019-12-13-countdown-to-the-new-year-ember-simple-auth.md
+responsive: true
+---
+
+**This is the 13th edition in our DecEmber series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 13
+
+Today we're going to look at [ember-simple-auth](https://github.com/simplabs/ember-simple-auth)!
```

Comment:
I think to fit the format of the other posts in the series, it' d be

```suggestion
Today we're going to look at [ember-simple-auth](https://emberobserver.com/addons/ember-simple-auth)!
```

What do you think?